src/styles/blocks/card.scss

```diff
@@ -0,0 +1,98 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  background-color: $white-color;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+  border-radius: 5px;
+  transition: 0.3s;
+
+  &:hover {
+    transform: scale(1.2);
```

Comment:
The hover effect scales the card to 1.2 times its size. This might cause layout issues if the card is close to other elements. Ensure there is enough space around the card to accommodate this transformation.